protocol/avdl/keybase1/favorite.avdl

```diff
@@ -33,6 +38,9 @@ protocol favorite {
     array<User> resetMembers;
 
     union { null, Time } mtime;
+
+    // Is the local view of this folder hosed due to stuck CR?
+    FolderConflictType conflictType;
```

Comment:
Hmm this `Folder` type is for both RPC param to reference a TLF, and as result from RPC to describe a TLF's metadata. It'd be nice if we can at some point separate the two as it doesn't make much sense to have resetMembers, conflict, etc. when referring to a TLF from GUI.

Reply:
Noted, feel free to make a ticket if it's important to you.  Seems like it would require some JS-side work I'm not comfortable doing.

Reply:
Cool; made KBFS-4157